### DISCUSSION TITLE
All values for $mail->

### DISCUSSION THREAD

**Discussion by Mazurky** [PHPMailer/PHPMailer#2597] [Q&A] [answered]
Is there somewhere wiki for all values for $mail-> /*this values*/
etc. $mail->isSMTP();
$mail->AuthType();

**Comment by rslenshub** [comment, marked as answer]
Maybe you are looking for this?

https://phpmailer.github.io/PHPMailer/classes/PHPMailer-PHPMailer-PHPMailer.html
